testing/util/sync_aggregate.go

```diff
@@ -10,15 +10,10 @@ import (
 	"github.com/prysmaticlabs/prysm/crypto/bls"
 	"github.com/prysmaticlabs/prysm/encoding/bytesutil"
 	ethpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1"
-	"github.com/prysmaticlabs/prysm/runtime/version"
 	"github.com/prysmaticlabs/prysm/time/slots"
 )
 
-func generateSyncAggregate(bState state.BeaconState, privs []bls.SecretKey, parentRoot [32]byte) (*ethpb.SyncAggregate, error) {
-	st, ok := bState.(state.BeaconStateAltair)
-	if !ok || bState.Version() == version.Phase0 {
-		return nil, errors.Errorf("state cannot be asserted to altair state")
-	}
```

Comment:
This check is irrelevant here because the interfaces are equivalent and the beacon state iface will have altair specific getters. if something is unsupported, it will throw an error